README.md

```diff
@@ -11,6 +12,61 @@ You can reach the maintainers of this project at:
 - [Slack channel](https://kubernetes.slack.com/messages/sig-node)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
 
+ 
+## Building the operator
+
+Checkout the sources
+
+```bash
+$ git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator $GOPATH/src/github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+
+Update the `Makefile` and edit `IMAGE` and `REGISTRY` one will need those later to update the operator manifest (`image: $REGISTRY/$IMAGE`).
+
+## Manual deploy of the operator
+
+Checkout the sources
+
+```bash
+$ git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+Update the  `Makefile` with the a custom image built and configure the namespace where the operator should be deployed.
+
+The default CR will create the operand (NFD) in the `node-feature-discovery-operator` namespace,
+the CR can be edited to choose another namespace and image. See the `manifests/0700_cr.yaml` for the default values.
+
+```bash
+$ cd node-feature-discovery-operator
+$ make deploy
+```
+
+The operator will use the operand node-feature-discovery image built from: https://github.com/kubernetes-sigs/node-feature-discovery
+
+To uninstall the operator run 
+
+```bash
+$ make undeploy
+```
+
+To verify the correct working of NFD a e2e test can be run as well: 
+
+```bash
+$ make test-e2e
```

Comment:
Not about this README, but running `make test-e2e` fails with multiple build-time errors, for me at leat.